src/ImageDeleteModal.jsx

```diff
@@ -17,115 +17,98 @@ function sortTags(a, b) {
-                           {_("Delete tagged images")}
-                       </Button>
-                       <Button variant="link" onClick={Dialogs.close}>{_("Cancel")}</Button>
-                   </>}
-            >
-                <Stack hasGutter>
-                    { repoTags.length > 1 && <StackItem>{_("Multiple tags exist for this image. Select the tagged images to delete.")}</StackItem> }
-                    <StackItem isFilled>
-                        { repoTags.map(x => {
-                            return (
-                                <Checkbox isChecked={checkedTags.indexOf(x) > -1}
-                                          id={"delete-" + x}
-                                          aria-label={x}
-                                          key={x}
-                                          label={x}
-                                          onChange={checked => this.onValueChanged(x, checked)} />
-                            );
-                        })}
-                    </StackItem>
-                </Stack>
-                { repoTags.length > 2 && <Button variant="link" onClick={this.pickAll}>{_("select all")}</Button> }
-            </Modal>
-        );
-    }
-}
+    };
+
+    return (
+        <Modal isOpen
+                 position="top" variant="medium"
+                 onClose={Dialogs.close}
+                 title={cockpit.format(_("Delete $0"), repoTags ? repoTags[0] : "")}
+                 footer={<>
+                     <Button id="btn-img-delete" variant="danger" isDisabled={checkedTags.length === 0}
+                             onClick={() => handleRemoveImage(checkedTags, checkedTags.length === repoTags.length)}>
+                         {_("Delete tagged images")}
+                     </Button>
+                     <Button variant="link" onClick={Dialogs.close}>{_("Cancel")}</Button>
+                 </>}
+        >
+            <Stack hasGutter>
+                { repoTags.length > 1 && <StackItem>{_("Multiple tags exist for this image. Select the tagged images to delete.")}</StackItem> }
+                <StackItem isFilled>
+                    { repoTags.map(x => {
+                        return (
+                            <Checkbox isChecked={checkedTags.indexOf(x) > -1}
+                                        id={"delete-" + x}
+                                        aria-label={x}
+                                        key={x}
+                                        label={x}
+                                        onChange={checked => onValueChanged(x, checked)} />
+                        );
+                    })}
+                </StackItem>
+            </Stack>
+            { repoTags.length > 2 && <Button isDisabled={repoTags.length === checkedTags.length} variant="link" onClick={pickAll}>{_("select all")}</Button> }
+        </Modal>
+    );
+};
+
+export default ImageDeleteModal;
```

Comment:
I actually love named exports. Why did you change it? It's also unrelated change.

Reply:
Sure, I reverted it.